starlite/app.py

```diff
@@ -277,6 +278,8 @@ def __init__(
             type_encoders: A mapping of types to callables that transform them into types supported for serialization.
             websocket_class: An optional subclass of :class:`WebSocket <starlite.connection.websocket.WebSocket>` to use for
                 websocket connections.
+            configuration: An optional instance of :class:`AppConfig` <startlite.config.AppConfig> to replace group up initial
+                configurations. If passed in addition to kwargs related to configurations, those kwargs will be ignored.
```

Comment:
```suggestion
            config: An instance of :class:`AppConfig` <startlite.config.AppConfig>. If passed, configure the application with values from this object instead of kwargs.
```